logstash-core/lib/logstash/instrument/null_metric.rb

```diff
@@ -1,6 +1,16 @@
 # encoding: utf-8
+require "logstash/instrument/metric"
+
 module LogStash module Instrument
  class NullMetric
+   class NullTimedExecution
+     def stop
+     end
+   end
+
+   NULL_TIMED_EXECUTION_INSTANCE = NullTimedExecution.new
```

Comment:
I don't want the overhead of creating  a new instance on each call since it will return a noop object.